pkg/kgo/config.go

```diff
@@ -1037,6 +1039,12 @@ func TransactionTimeout(timeout time.Duration) ProducerOpt {
 	return producerOpt{func(cfg *cfg) { cfg.txnTimeout = timeout }}
 }
 
+// ConcurrentTransactionBackoff sets the backoff interval to use during
+// transactional requests in case we encounter CONCURRENT_TRANSACTIONS error.
+func ConcurrentTransactionBackoff(backoff time.Duration) ProducerOpt {
```

Comment:
Can this be moved below into a ConsumerOpt? Or perhaps above to just a plain Opt, since transactions kinda encompass both producing & consuming.

Reply:
Done